cmd/juju/machine/upgradeseries.go

```diff
@@ -80,23 +80,28 @@ func NewUpgradeSeriesCommand() cmd.Command {
 	return modelcmd.Wrap(&upgradeSeriesCommand{})
 }
 
-//go:generate go run github.com/golang/mock/mockgen -package mocks -destination mocks/upgradeMachineSeriesAPI_mock.go github.com/juju/juju/cmd/juju/machine UpgradeMachineSeriesAPI
+//go:generate go run github.com/golang/mock/mockgen -package mocks -destination mocks/upgrademachineseries_api_mock.go github.com/juju/juju/cmd/juju/machine UpgradeMachineSeriesAPI
 type UpgradeMachineSeriesAPI interface {
 	BestAPIVersion() int
 	Close() error
-	UpgradeSeriesValidate(string, string) ([]string, error)
 	UpgradeSeriesPrepare(string, string, bool) error
 	UpgradeSeriesComplete(string) error
 	WatchUpgradeSeriesNotifications(string) (watcher.NotifyWatcher, string, error)
 	GetUpgradeSeriesMessages(string, string) ([]string, error)
 }
 
+//go:generate go run github.com/golang/mock/mockgen -package mocks -destination mocks/status_api_mock.go github.com/juju/juju/cmd/juju/machine StatusAPI
```

Comment:
Since these come from and go to the same package, they could be a single file/statement.